CONTRIBUTING.md

```diff
@@ -1,70 +1,55 @@
 # Contributing to Docker
 
-Want to hack on Docker? Awesome! Here are instructions to get you
-started. They are probably not perfect; please let us know if anything
-feels wrong or incomplete.
+Want to hack on Docker? Awesome!  We have a contributor's guide that explains [setting up a Docker development environment and the contributing process](https://docs.docker.com/project/who-written-for/). 
+
+[![Foo](docs/sources/static_files/contributors.png)](https://docs.docker.com/project/who-written-for/)
+
+This page contains information about reporting issues as well as some tips and guidelines useful to experienced open source contributors. Finally, make sure you read our [community guidelines](#docker-community-guidelines) before you start participating.
 
 ## Topics
 
 * [Reporting Security Issues](#reporting-security-issues)
 * [Design and Cleanup Proposals](#design-and-cleanup-proposals)
-* [Reporting Issues](#reporting-issues)
-* [Build Environment](#build-environment)
-* [Contribution Guidelines](#contribution-guidelines)
+* [Reporting Issues](#reporting-other-issues)
+* [Quick Contribution Tips and Guidelines](#quick-contribution-tips-and-guidelines)
 * [Community Guidelines](#docker-community-guidelines)
 
-## Reporting Security Issues
-
-The Docker maintainers take security very seriously. If you discover a security issue,
-please bring it to their attention right away!
-
-Please send your report privately to [security@docker.com](mailto:security@docker.com),
-please **DO NOT** file a public issue.
-
-Security reports are greatly appreciated and we will publicly thank you for it. We also
-like to send gifts - if you're into Docker shwag make sure to let us know :)
-We currently do not offer a paid security bounty program, but are not ruling it out in
-the future.
+## Reporting security issues
 
-## Design and Cleanup Proposals
+The Docker maintainers take security seriously. If you discover a security
+issue, please bring it to their attention right away!
 
-When considering a design proposal, we are looking for:
+Please **DO NOT** file a public issue, instead send your report privately to
+[security@docker.com](mailto:security@docker.com), 
 
-* A description of the problem this design proposal solves
-* A pull request, not an issue, that modifies the documentation describing
-  the feature you are proposing, adding new documentation if necessary.
-  * Please prefix your issue with `Proposal:` in the title
-* Please review [the existing Proposals](https://github.com/docker/docker/pulls?q=is%3Aopen+is%3Apr+label%3AProposal)
-  before reporting a new one. You can always pair with someone if you both
-  have the same idea.
+Security reports are greatly appreciated and we will publicly thank you for it.
+We also like to send gifts &mdash; if you're into Docker schwag make sure to let
+us know :) We currently do not offer a paid security bounty program, but are not
```

Comment:
The emoticon was in the original. I was trying to preserve the original tone. I'll take it out.   The semi-colon is part of the HTML code. Doesn't exist in the actual text.